storage-shared-benchmarking/src/main/java/com/google/cloud/storage/benchmarking/W1R3.java

```diff
@@ -58,37 +58,39 @@ final class W1R3 implements Callable<String> {
   @Override
   public String call() throws Exception {
     // Create the file to be uploaded and fill it with data
-    TmpFile file = DataGenerator.base64Characters().tempFile(tempDirectory, objectSize);
-    BlobInfo blob = BlobInfo.newBuilder(bucketName, file.toString()).build();
+    try (TmpFile file = DataGenerator.base64Characters().tempFile(tempDirectory, objectSize)) {
```

Comment:
you don't want to use the base64 character set here for generate. It creates highly compressible and dedupeable data which Gcs sometimes will optimize for leading to skewed data.

You can either use the existing `rand` or port the `devUrandom` implementation from the existing benchmarks into this repos version.

it's fine to do this in a follow up pr.